students/david/src/01-fibonacci/iterativeFibonacci.ts

```diff
@@ -0,0 +1,37 @@
+const iterativeFibonacci = (number:number):bigint => {
```

Comment:
missing docs. Public API should be documented using JSDoc or somethin similar. The code should be selfcommented